lib/glific_web/flows/flow_editor_controller.ex

```diff
@@ -229,6 +229,13 @@ defmodule GlificWeb.Flows.FlowEditorController do
     json(conn, %{results: results})
   end
 
+  @spec get_language_map :: map()
+  defp get_language_map,
```

Comment:
```suggestion
  defp get_language_id_local_map,
```

Reply:
Let's move this function to lib/glific/settings.ex only like another language-specific functions.